pkg/controllers/cloud/networkpolicy.go

```diff
@@ -948,17 +1029,27 @@ func (a *appliedToSecurityGroup) notify(op securityGroupOperation, status error,
 	case securityGroupOperationUpdateMembers:
 		a.hasMembers = true
 	case securityGroupOperationUpdateRules:
-		// AppliedToSecurityGroup added rules, now add members.
+		// AppliedToSecurityGroup added rules, now update addrGroup references and add members.
```

Comment:
Generally, at level comments like this are used for function to share the intent. this is fine for now. but in future if this case grows, this kind of top level comments will go thru iterations